lang/nqp/Portfile

```diff
@@ -41,11 +41,14 @@ legacysupport.redirect_bins nqp nqp-m
 configure.cmd       ${prefix}/bin/perl Configure.pl
 configure.args      --prefix=${prefix} \
                     --backends=moar \
-                    --with-moar=${prefix}/bin/moar
+                    --with-moar=${prefix}/bin/moar \
+                    --no-silent-build
 
 # https://github.com/MoarVM/MoarVM/issues/414
 universal_variant   no
 # Unsupported by Configure.pl
 # configure.universal_args-delete --disable-dependency-tracking
 
+use_parallel_build  no
```

Comment:
Just curious... are we 100% sure parallel builds can't be fixed...?

Reply:
> My system froze during the build, so I decided to keep that line from the @barracuda156's [portfile](https://github.com/macos-powerpc/powerpc-ports/blob/cfefe5b0831bd7d752a39a6f6c4a746e03349017/lang/nqp/Portfile#L52).

Right, but does it also freeze on the Ci builders, when parallel builds are enabled?

Let's not assume it's a problem with the port, until we've got more information.

Reply:
> My system froze during the build, so I decided to keep that line from the @barracuda156's [portfile](https://github.com/macos-powerpc/powerpc-ports/blob/cfefe5b0831bd7d752a39a6f6c4a746e03349017/lang/nqp/Portfile#L52).

Update: while my system didn't freeze, the build did fail. And since nqp seems to build relatively quickly regardless, I'm good with keeping parallel builds disabled for now.